File: 프로그래머스/lv1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.py
def solution(s):
    answer = True
    a=s.upper()
    
    if a.count('P') != a.count('Y'):
        answer =False
    else:
        answer =True
    if a.count('P') == 0 and a.count('Y')==0:
        answer = True

    return answer

File: 프로그래머스/1/12916. 문자열 내 p와 y의 개수/문자열 내 p와 y의 개수.java
class Solution {
    boolean solution(String s) {
        boolean answer = true;
        s = s.toUpperCase();
        
        answer = s.chars().filter(e->'P'==e).count() == s.chars().filter(e->'Y'==e).count();
        

        return answer;
    }
}
